packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts

```diff
@@ -246,3 +246,79 @@ describe('va-pagination', () => {
   });
 
 });
+
+describe('va-pagination - uswds', () => {
+  it.skip('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-pagination></va-pagination>');
+
+    const element = await page.find('va-pagination');
+    expect(element).toEqualHtml(`
+      <va-pagination page="1" pages="2" max-page-list-length="2" uswds class="hydrated">
+        <mock:shadow-root>
+          <nav class="usa-pagination">
+            <ul class="usa-pagination__list">
+              <li class="usa-pagination__item usa-pagination__page-no">
+                <a href="javascript:void(0)" class="usa-pagination__button usa-current">1</a>
+              </li>
+              <li class="usa-pagination__item usa-pagination__page-no">
+                <a href="javascript:void(0)" class="usa-pagination__button">2</a>
+              </li>
+              <li class="usa-pagination__item usa-pagination__arrow" aria-label="Next page">
+                <a href="javascript:void(0)" class="usa-pagination__link usa-pagination__next-page">
+                  <span class="usa-pagination__link-text">Next</span>
+                  <div id="next-icon-arrow"></div>
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </mock:shadow-root>
+      </va-pagination>
+    `);
+  });
+
+  it('only selected "page" has selected class', async () => {
```

Comment:
If all of these added tests are specifically for the `uswds` variation, then can you add "uswds" to the front of the test name? Like this: https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-number-input/test/va-number-input.e2e.ts#L318


That is the pattern we have been using for all of the other uswds tests. Since both v1 and v3 variations live in this test file, it makes it easier to distinguish.